### PR TITLE
feat: make ripple color dynamic for ListAccordion

### PR DESCRIPTION
**Pull request by suleymanbariseser**
### Summary

It makes rippleColor dynamic in ListAccordion as it is asked in the issue #3439 

### Test plan

Tested in the example application with different text colors and everything seems okay.

### CODE REVIEW

File: src/components/List/ListAccordion.tsx [hunk 1]
```diff
@@ -187,6 +187,11 @@ const ListAccordion = ({
   const isExpanded = groupContext
     ? groupContext.expandedId === id
     : expandedInternal;
+
+  const titleTextColor = isExpanded ? theme.colors?.primary : titleColor;
+
+  const rippleColor = color(titleTextColor).alpha(0.12).rgb().string();
```
Comment (lukewalczak) [inline src/components/List/ListAccordion.tsx:193]: Could you please add a **unit** test covering `rippleColor`?

Comment (suleymanbariseser) [reply src/components/List/ListAccordion.tsx:193]: sure but i think to test it I need to create a utils file and move it there, right? Additionally, I saw tests are missing for titleColor and descriptionColor so if it is okay I can add the tests for them too

Comment (suleymanbariseser) [reply src/components/List/ListAccordion.tsx]: @lukewalczak tests are added

**Review by lukewalczak — APPROVED** [review]
Thanks @suleymanbariseser!

**Pull request merged**